src/backend/Texture.h

```diff
@@ -40,6 +40,7 @@ class Texture : public cocos2d::Ref
     TextureFormat _textureFormat = TextureFormat::R8G8B8;
     TextureUsage _textureUsage = TextureUsage::READ;
     bool _isMipmapEnabled = false;
+    uint32_t _textureHashCode = 0;
```

Comment:
It seems `_textureHashCode ` is not used.

Reply:
It use in TextureMTL::updateData to skip duplicate operation.